pypowsybl/report.py

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2022, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+from pypowsybl._pypowsybl import create_reporter_model, print_report, JavaHandle
+
+class Reporter:
+    def __init__(self, task_key: str = '', default_name: str = ''):
+        self.reporter_model_handle = create_reporter_model(task_key, default_name)
+
+    def __repr__(self) -> str:
+        return print_report(self.reporter_model_handle)
+
+    @property
+    def reporter_model(self) -> JavaHandle:
```

Comment:
We should not expose to the user the java handle, which is an implementation detail --> let's prefix it with an underscore as we do in other places (see network for ex.)

Reply:
done